docs/source/notes/gradcheck.rst

```diff
@@ -0,0 +1,218 @@
+.. _gradcheck-mechanics:
+
+Gradcheck mechanics
+===================
+
+This note presents an overview of how the :meth:`~torch.autograd.gradcheck` and :meth:`~torch.autograd.gradgradcheck` functions work.
+
+It will cover both forward and backward mode AD as well as complex valued functions and higher order derivatives.
+
+.. contents:: :local:
+    :depth: 2
+
+Notations and background informations
+-------------------------------------
+
+Throughout this note, we will use :math:`x`, :math:`y`, :math:`a`, :math:`b`, :math:`v`, :math:`u`, :math:`ur` and :math:`ui` as real valued vectors and :math:`z` is a complex valued vector.
```

Comment:
```suggestion
Throughout this note, we will use :math:`x`, :math:`y`, :math:`a`, :math:`b`, :math:`v`, :math:`u`, :math:`ur` and :math:`ui` to represent real valued vectors and :math:`z` to represent a complex valued vector.
```

Reply:
I don't this comment is addressed. could you consolidate all the conventions and present it in a list? I think it would make it easier for the readers of this note to read/come back and check the conventions, if needed, at any point.